icu4j/maven-build/extract-data-files.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
```

Comment:
Nitpick: to be complete should also have a Windows script.

Reply:
This script is just to simplify `cldr-icu-readme.txt`, which is a set of steps that need to be run by a person (who can double check discrepencies, etc.), and those steps are written for a POSIX on *nix environment. So I think this is okay for now (until there exists an easy cross-platform solution to do things).